src/backend/joanie/tests/core/api/order/test_submit_installment_payment.py

```diff
@@ -405,7 +405,7 @@ def test_api_order_submit_installment_payment_without_passing_credit_credit_card
             },
         )
 
-        self.assertTrue(_mock_create_payment.called)
+        self.assertTrue(mock_create_payment.called)
```

Comment:
This is redundant, you are already testing that is method has been called with `assert_called_once_with`

```suggestion
```